### ISSUE TITLE
Note to translators about unsupported languages.

### ISSUE THREAD

**Issue by FreeCAD-Tools** [shaise/FreeCAD_FastenersWB#209]
While I was figuring out the translations, I noticed Mexican Spanish 
https://github.com/shaise/FreeCAD_FastenersWB/blob/master/translations/fasteners_es_mx.ts
is not support in FreeCAD.

This may checked by command (https://wiki.freecadweb.org/Localisation):
  
      FreeCADGui.supportedLocales()

      {'English': 'en', 'Afrikaans': 'af', 'Arabic': 'ar', 'Basque': 'eu', 'Bulgarian': 'bg', 
      'Catalan': 'ca', 'Chinese Simplified': 'zh-CN', 'Chinese Traditional': 'zh-TW', 
      'Croatian': 'hr', 'Czech': 'cs', 'Dutch': 'nl', 'Filipino': 'fil', 'Finnish': 'fi', 'French': 'fr', 
      'Galician': 'gl', 'Georgian': 'ka', 'German': 'de', 'Greek': 'el', 'Hungarian': 'hu', 
      'Indonesian': 'id', 'Italian': 'it', 'Japanese': 'ja', 'Kabyle': 'kab', 'Korean': 'ko', 
      'Lithuanian': 'lt', 'Norwegian': 'no', 'Polish': 'pl', 'Portuguese': 'pt-PT', 
      'Portuguese, Brazilian': 'pt-BR', 'Romanian': 'ro', 'Russian': 'ru', 'Slovak': 'sk', 
      'Slovenian': 'sl', 'Spanish': 'es-ES', 'Spanish, Argentina': 'es-AR', 'Swedish': 'sv-SE', 
      'Turkish': 'tr', 'Ukrainian': 'uk', 'Valencian': 'val-ES', 'Vietnamese': 'vi'}

In other words, the translation is exist, but user cannot select it in FreeCAD settings.
It is desirable that users make translations for languages used in FreeCAD.

**Comment by shaise** [comment]
This should be a FreeCad issue, not Fasteners.

**Comment by hasecilu** [comment]
Hi, I translated that file, you are correct there are only 2 selections for Spanish in FreeCAD, Spanish from Spain and Argentina.

Since it will not be used I think we can delete those files and I can check the es-es and es-ar files, I don't speak those variants but it's okay.

**Comment by hasecilu** [comment]
@shaise 
With #211 merged I think we can close this

**Comment by FreeCAD-Tools** [comment]
@hasecilu
Thanks. 

Even if delete a file from the repository, it still remains in its memory (history). In general, the work you have done will still remain in the project and it can be restored if desired, from this link (If Mexican Spanish language is be added to FreeCAD):

https://github.com/shaise/FreeCAD_FastenersWB/tree/d90c268ffc83c9a6e9e309e686faf321bbb299f3/translations

**Comment by hasecilu** [comment]
> Even if delete a file from the repository, it still remains in its memory (history). In general, the work you have done will still remain in the project and it can be restored if desired, from this link (If Mexican Spanish language is be added to FreeCAD):

Yep, I prefer to stick to one already supported language because starting another from zero would require a huge effort, and still with the ones  that already exist there are a lot of checks to do to guarantee coherence, good grammar, etc

**Comment by luzpaz** [comment]
So the gist here is that there is one dialect (is that the correct term) of spanish that is not supported by upstream FreeCAD ?

**Comment by FreeCAD-Tools** [comment]
Yes.

Just before doing a translation, translator need to check if the language is supported in FreeCAD. If language is not supported in FreeCAD "core", then there is no point in translation. It will simply take up space in the workbench repository.

**Comment by FreeCAD-Tools** [comment]
I think this issue can be closed, because the information is was duplicated at the workbench description.
https://github.com/shaise/FreeCAD_FastenersWB/pull/250

**Comment by kaktusus** [comment]
A developer's manual was created:

https://github.com/FreeCAD/DevelopersHandbook/blob/master/technical/translation.md
